cmd/tailscale/main.go

```diff
@@ -602,6 +598,20 @@ func (a *App) login(ctx context.Context) {
 	}
 }
 
+func (a *App) logout(ctx context.Context) error {
+	ctx, cancel := context.WithDeadline(ctx, time.Now().Add(5*time.Minute))
+	defer cancel()
+	r, err := a.localAPIClient.Call(ctx, "POST", "logout", nil)
+	defer r.Body().Close()
+
+	if err != nil {
+		log.Printf("logout: %s", err)
+		a.backend.Logout(ctx)
```

Comment:
If localAPIClient.Call() timed out, this will immediately fail with a timeout as well. Maybe create a new context with its own context?

Reply:
Sounds good. Using the 5 minute timeout that was originally used for the logout timeout here, and using a shorter timeout for the localapi call.